roles/add-vhost/tasks/main.yml

```diff
@@ -40,3 +45,11 @@
 - name: Add custom domain
   shell: "drush @stanfordpro.prod ac-domain-add {{ new_absolute_path }}"
   when: new_absolute_path != "" and launch_tasks == "launch"
+
+# Force loading through the canonical URL by setting the
+# "stanfordpro_helper_canonical_url" variable to (e.g.) "foo.stanford.edu". This
+# will only take affect on sites on the prod environment, because the
+# stanfordpro_helper module restricts that functionality to prod
+- name: Set stanfordpro_helper_canonical_url variable prior to launch
+  shell: "{{ drush_alias }} -y --exact vset stanfordpro_helper_canonical_url {{ new_absolute_path }}"
```

Comment:
I am making the assumption that we plan to add a module to do something with this.

Reply:
I too am making the same assumption.